integration-tests/prepare.sh

```diff
@@ -78,7 +78,7 @@ function index_all_tags() {
 function clone_tendermint_config() {
     print_step "Cloning Tendermint config from \"${1}\" to \"${2}\""
 
-    rm -rf "${2}"
+    sudo rm -rf "${2}"
```

Comment:
in travis, it is no problem, because it's root.
so i added sudo for these commands.

Reply:
why is sudo required?

Reply:
because in running,
./prepare.sh
it makes root owned file inside docker,
so external shell is normal user,
it cannot continue

Reply:
i didn't execute as root and it worked fine

Reply:
@calvinaco @calvinlauco is sudo really required?

Reply:
which folder did you run the script?
for me, i ran inside 
cd integrations-tests
./prepare.sh

Reply:
doing sudo also seems overkill; maybe after each generated docker-data, try to change the ownership to the current user?

Reply:
agree, i will try different approach